### DISCUSSION TITLE
How to blend (append) one image below the other so that the border where they are attached looks blended in

### DISCUSSION THREAD

**Discussion by srini8080** [ImageMagick/ImageMagick#6439] [Digital Image Processing]
IM7
Centos stream 8

I have 2 images. I want to append one image below the other. However, when I append them, I will get the edge where it is attached as a line (enclosed). I want 20 pixels on both sides smudged in such a way that it looks natural progress on the top page into the bottom page

![22_1](https://github.com/ImageMagick/ImageMagick/assets/12046302/c83ea3db-41b8-4372-bad9-2dbe37f3ebd2)
![SEC_text_0](https://github.com/ImageMagick/ImageMagick/assets/12046302/11f0df41-3ac8-4397-bbc8-dc4b1cf18989)
![SEC_page0](https://github.com/ImageMagick/ImageMagick/assets/12046302/463e1c98-9de9-416b-b49f-b0c0814d6626)

Any help is appreciated

Thanks

**Comment by snibgo** [comment]
You could do an "alpha blend". This will overlap the two images by 20 rows.

The images are both 1240 high, so appended the result would be 2480 high, but with an overlap of 20 the final height will be 2460. I extend the first to the new height, then compose the two inputs using a mask that is black at the top and white at the bottom, with a transition where the overlap occurs. Windows BAT script:
```
magick ^
  ( blend1.png -extent x2460 ) ^
  ( blend2.png ) ^
  ( -size 1754x1220 xc:Black ^
    -size 1754x20 gradient:Black-White ^
    -size 1754x1220 xc:White ^
    -append +repage ^
  ) ^
  -gravity South ^
  -alpha off ^
  -composite ^
  x.png
```

**Reply by srini8080** [reply]
I put it into unix script 

magick \( "$image1" -extent x2460 \) \( "$image2" \) \( -size 1754x1220 xc:Black -size 1754x20 gradient:Black-White -size 1754x1220 xc:White \
    -append +repage \) -gravity South -alpha off -composite x.png

It gave this output

Thanks.

![x](https://github.com/ImageMagick/ImageMagick/assets/12046302/6b2acb68-ea0f-40d7-bb73-cf4a6e1b6ff2)

**Reply by srini8080** [reply]
sorry, my bad. It worked.

Thanks
